optbinning/binning/binning.py

```diff
@@ -616,8 +616,8 @@ def transform(self, x, metric="woe", metric_special=0,
 
         Returns
         -------
-        x_new : numpy array, shape = (n_samples,)
-            Transformed array.
+        x_new : numpy.ndarray
```

Comment:
Throughout the library, I try to follow Scikit-learn's docstring style. See for example: https://github.com/scikit-learn/scikit-learn/blob/2beed5584/sklearn/linear_model/_logistic.py#L1283